pkgs/applications/graphics/feh/default.nix

```diff
@@ -1,29 +1,35 @@
-{ stdenv, fetchurl, makeWrapper, xorg, imlib2, libjpeg, libpng
+{ stdenv, fetchurl, makeWrapper
+, xorg, imlib2, libjpeg, libpng
 , curl, libexif, perlPackages }:
 
+with stdenv.lib;
+
 stdenv.mkDerivation rec {
-  name = "feh-2.18.1";
+  name = "feh-${version}";
+  version = "2.18.2";
```

Comment:
The version field doesn't seem to be used anywhere else so simpler to just leave it directly in the name.